mlrun/config.py

```diff
@@ -464,11 +464,20 @@
         "expose_internal_api_endpoints": False,
     },
     "default_workflow_runner_name": "workflow-runner-{}",
-    "sidecar": {
-        "log_collector": {
-            "address": "",
-            "enabled": False,
-        },
+    "log_collector": {
+        "address": "",
```

Comment:
Shouldn't we put here the default sidecar address? Something like `localhost:<some port>`?

Reply:
yeah why not